COPYRIGHT

```diff
@@ -0,0 +1,5 @@
+MedTech CoRE Doctoral Training Programme Visualisation Tutorial
```

Comment:
Should probably be something like "MedTech CoRE Doctoral Training Programme Computational Physiology Module (visualisation component)"

or perhaps: "This visualisation tutorial is part of the MedTech CoRE Doctoral Training Programme's Computational Physiology module"